config/routes.rb

```diff
@@ -13,12 +13,13 @@
   mount SwaggerUiEngine::Engine, at: '/open-api'
 
   def add_swagger_route http_method, path, opts = {}
+    prefix = "api"
+    prefix = "#{ENV["PATH_PREFIX"]}/#{ENV["APP_NAME"]}" if ENV["PATH_PREFIX"].present? && ENV["APP_NAME"].present?
```

Comment:
@carbonin Should we use a File.join and to_s to account for leading trading slashes in the 2 env's?